### PR TITLE
Add cloudflare cache clear script

### PR DESCRIPTION
**Pull request by tuchfarber**

### CODE REVIEW

File: tubular/scripts/purge_cloudflare_cache.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env python3
+
+"""
+Command-line script to deploy a frontend app to s3.
```
Comment (albemarle) [inline tubular/scripts/purge_cloudflare_cache.py]: Can you update this?

**Review by albemarle — APPROVED** [review]

**Comment by tuchfarber** [comment]
Tests were failing from code in #392 . Fixed in a separate commit of this PR.

**Review by adzuci — APPROVED** [review]
:shipit:

**Comment by adzuci** [comment]
> Tests were failing from code in #392 . Fixed in a separate commit of this PR.

Sorry for not catching those failed tests =/.

**Pull request merged**